releases/release-1.8/README.md

```diff
@@ -0,0 +1,43 @@
+# Kubeflow 1.8
+
+### Links
+
+- [Release Team](release-team.md)
+- [Calendar](https://arrik.to/kf-release-team-cal)
+- [Meeting Notes](https://arrik.to/kf-release-team-notes)
+- [Recordings](https://arrik.to/kf-release-team-recordings)
+- [Retrospective Document](https://bit.ly/kf-release-1-6-retro)
+- Project Board
+- Contact
+  - [#release](https://app.slack.com/client/T7QLHSH6U/C9V2WT2KV) on slack
+  - [@release-team](https://github.com/orgs/kubeflow/teams/release-team) on GitHub
+
+### TL;DR
+
+The 1.8 release cycle is proposed as follows
+
+- **Friday, May 5th 2023**     : Week 0      - Release team is formed
+- **Monday, May 8th 2023**     : Week 1  C\* - Release cycle begins
+- **Friday, May 12th 2023**    : Week 1  C\* - Roadmaps are finalized
+- **Wednesday, Aug 16th 2023** : Week 15 C\* - Feature freeze
+- **Wednesday, Aug 21st 2023** : Week 16     - Manifests testing and bug fixing starts
+- **Friday, Sept 1st 2023**    : Week 17 C\* - Manifests testing and bug fixing ends
+- **Monday, Sept 4th 2023**    : Week 18     - Docs update starts
+- **Monday, Sept 4th 2023**    : Week 18     - Distribution testing starts
+- **Friday, Sept 22nd 2023**   : Week 20     - Distribution testing ends
+- **Monday, Sept 25th 2023**   : Week 21 C\* - Pre-release bug fixing stage starts
+- **Friday, Oct 6th 2023**     : Week 22     - Pre-release bug fixing stage ends
+- **Friday, Oct 6th 2023**     : Week 22     - Docs update ends
+- **Wednesday, Oct 11th 2023** : Week 23 C\* - Kubeflow v1.8 Released
```

Comment:
If the docs are published from the `master` branch, they must correspond to the latest release. Shouldn't we update documentation at the same time with the release?

Reply:
Ideally the content should be ready before the release, that way on release date we just have to publish.
@annajung can you confirm this information?

Reply:
It's not a strong opinion. As a user I expect the documentation to reflect the latest available release. To achieve this, docs should be released together with the actual release, or right after it.

Reply:
I also think it makes sense to be closer to the release date. Maybe a day before should be the last day to merge docs, and website release process can be done during release day